### PR TITLE
scrcpy 1.13

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scrcpy.rb [hunk 1]
```diff
@@ -17,8 +17,8 @@ class Scrcpy < Formula
   depends_on "sdl2"
 
   resource "prebuilt-server" do
-    url "https://github.com/Genymobile/scrcpy/releases/download/v1.12.1/scrcpy-server-v1.12.1"
-    sha256 "63e569c8a1d0c1df31d48c4214871c479a601782945fed50c1e61167d78266ea"
+    url "https://github.com/Genymobile/scrcpy/releases/download/v1.13/scrcpy-server-v1.13"
```
Comment (SMillerDev) [inline Formula/scrcpy.rb:20]: I assume these always need to be in sync, maybe we can use `#{version}` here instead?

Comment (alebcay) [reply Formula/scrcpy.rb:20]: That's a good point. I don't recall any time that these have fallen out of sync.

**Comment by Bo98** [comment]
`#{version}` doesn't work in resource blocks it seems:

```
 ==> Downloading https://github.com/Genymobile/scrcpy/releases/download/v/scrcpy-server-v
curl: (22) The requested URL returned error: 404 Not Found
```

**Comment by alebcay** [comment]
Switched back to not using `#{version}`.

**Pull request closed without merging**